tidb-server/main.go

```diff
@@ -520,7 +531,7 @@ func setupTracing() {
 	tracingCfg := cfg.OpenTracing.ToTracingConfig()
 	tracer, _, err := tracingCfg.New("TiDB")
 	if err != nil {
-		log.Fatal("cannot initialize Jaeger Tracer", err)
+		log.Fatal("setup trace", zap.String("cannot initialize jaeger tracer", err.Error()))
```

Comment:
`log.Fatal("setup jaeger tracer failed", zap.String("error message", err.Error()))` will be more better?

Reply:
Done.